Assets/Player/InteractionContexts/SelectionContext.gd

```diff
@@ -62,8 +62,9 @@ func _on_ia_alt_command_released(target: Node, position: Vector3) -> void:
 	var top_left: Vector2 = get_viewport().get_camera().unproject_position(sel_pos_start)
 	var bottom_right: Vector2 = get_viewport().get_camera().unproject_position(sel_pos_end)
 	if top_left.distance_to(bottom_right) <= selection_tolerance:
-		if target.is_in_group("units"):
-			selection = [target]
+		if typeof(target) == TYPE_OBJECT:	
```

Comment:
```suggestion
```
Does the check actually fail at some point? Unless there is a case it can happen, there is no point to add complexity right now. Also, since this is unrelated to the health/destruct PR, it belongs to its own PR.

Reply:
When I was testing the project on my system, when I would try to select godot would throw an error about trying to do something on a base nil. I was on the World map and it happened whenever I would try to select since there was nothing on that map. I do understand about putting this in it's own commit/PR though since it is unrelated to the other things.

Reply:
I see. The World map is actually pretty old and wasn't updated since (it's just a simple test map for rotation and sprite placement) and the method above relies on a raycast to a plane which isn't existing there, hence the error. `WorldTown.tscn` represents the actual gameplay and as long as it works there, it should be fine.